docs/capnp.rst

```diff
@@ -14,36 +14,6 @@ Classes
 RPC
 ~~~
 
-Promise
-#######
-
-.. autoclass:: Promise
-  :members:
-  :undoc-members:
-  :inherited-members:
-
-Promise may be one of:
-
-* :meth:`capnp.lib.capnp._Promise`
-* :meth:`capnp.lib.capnp._RemotePromise`
-* :meth:`capnp.lib.capnp._VoidPromise`
-
-.. autoclass:: capnp.lib.capnp._Promise
-  :members:
-  :undoc-members:
-  :inherited-members:
-
-.. autoclass:: capnp.lib.capnp._RemotePromise
-  :members:
-  :undoc-members:
-  :inherited-members:
```

Comment:
Most of this can indeed be removed. But `_RemotePromise` still has the purpose of pipelining. That should be documented somewhere I guess (not necessarily here).

Reply:
I readded `_RemotePromise`. Although its code documentation does not really help with the pipelining ... However there is a section in the quickstart that explains this. It is quite short but with your latest adaptions the usage has become quite straightforward 😇